src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -108,6 +108,16 @@ impl ScalarFunc {
         self.children[0].eval_int(ctx, row)
     }
 
+    #[inline]
+    pub fn round_dec(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
+        let d = try_opt!(self.children[0].eval_decimal(ctx, row));
+        let d: Result<Decimal> = d
+            .into_owned()
+            .round(DEFAULT_FSP, RoundMode::HalfEven)
+            .into();
+        d.and_then(|dec| dec.as_i64_with_ctx(ctx)).map(Some)
```

Comment:
I think we need to return decimal according to https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_round

> The return value has the same type as the first argument (assuming that it is integer, double, or decimal). 

and the related function in tidb https://github.com/pingcap/tidb/blob/master/expression/builtin_math.go#L366

Reply:
@AndreMouche One more question, does it need second parameter D in tikv?
> ROUND(X), ROUND(X,D)
>
> Rounds the argument X to D decimal places. The rounding algorithm depends on the data > type of X. D defaults to 0 if not specified. D can be negative to cause D digits left of the >decimal point of the value X to become zero.

Reply:
@AndreMouche Oh, it seems ROUND(X) are RoundWithFracXXX